src/pages/developers/standards/_meta.json

```diff
@@ -0,0 +1,10 @@
+{
+  "nft": {
+    "title": "Universal NFT",
+    "description": "Universal NFT."
```

Comment:
Can we elaborate on what a 'Universal NFT' means? Users will most likely know what an NFT is, but what makes a Universal NFT different?

Reply:
I believe this piece from `/developers/standards/nft` works better as a brief description in the context of a card:

> The Universal NFT standard enables non-fungible tokens (ERC-721 NFT) to be minted on any chain and seamlessly transferred between connected chains.